.github/workflows/test.yml

```diff
@@ -44,40 +94,30 @@ jobs:
         uses: actions/setup-python@v5
         with:
           python-version: '3.12'
+      - name: Install the latest version of uv
+        uses: astral-sh/setup-uv@v5
       - name: Install ANTs
         run: |
-          conda install -c conda-forge ants=2.5
-      - name: Install datalad
+          # Override libiconv pre-installed from anaconda channel
+          # See https://github.com/conda-forge/libitk-feedstock/issues/98
+          # Since we're not creating a new environment, we must be explicit
+          conda install -c conda-forge ants=2.5 libiconv
+      - name: Verify antsRegistration path
         run: |
-          python -m pip install datalad-installer
-          datalad-installer --sudo ok git-annex
-          python -m pip install datalad datalad-osf
-          datalad wtf
+          export PATH=$ANTSPATH:$PATH
+          which antsRegistration
+          antsRegistration --version
       - uses: actions/cache/restore@v4
         with:
           path: /home/runner/nifreeze-tests/
-          key: data-v0
-      - name: Get test data with DataLad
-        run: |
-          if [[ ! -d "${TEST_DATA_HOME}" ]]; then
-            datalad install -rg --source=https://gin.g-node.org/nipreps-data/tests-nifreeze.git ${TEST_DATA_HOME}
-          else
-            cd ${TEST_DATA_HOME}
-            datalad update --merge -r .
-            datalad get -r -J4 *
-          fi
-      - uses: actions/cache/save@v4
-        with:
-          path: /home/runner/nifreeze-tests/
-          key: data-v0
+          key: data-v1-${{ needs.cache-test-data.outputs.SHA }}
       - name: Install TeX Live
         run: |
           sudo apt-get update
           sudo apt install texlive texlive-latex-extra texlive-fonts-recommended cm-super dvipng
       - name: Install tox
         run: |
-          python -m pip install --upgrade pip
-          python -m pip install tox tox-gh-actions
+          uv tool install --with=tox-uv --with=tox-gh-actions tox
```

Comment:
The switch to `uv tool` instead of `pip` was an experiment in trying to figure out what in the environment caused the tests to fail. I could revert it, but I'll plan to do this in another PR anyway, so I'll leave it unless asked.